src/utility/api.tsx

```diff
@@ -0,0 +1,36 @@
+type Task = {
+  task: {
+    status: string;
+    id: number;
+  };
+};
+
```

Comment:
This files needs to be rewrite
Add this to prevent ESLint to drive you crazy and not allowing you to commit
```
/* eslint-disable */
```